### PR TITLE
[12.0][MIG] connector ecommerce

### PR DESCRIPTION
**Pull request by ljimenezsidoo**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Runbot status

File: connector_ecommerce/views/sale_view.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    type="object" class="oe_highlight"
+                    attrs="{'invisible': [('parent_id', '=', False)]}"
+                    groups="base.group_user" />
+            </xpath>
+            <xpath expr="//header" position="inside">
+                <button name="%(action_sale_ignore_cancel)d"
+                    type="action"
+                    attrs="{'invisible': ['|', '|', ('canceled_in_backend', '=', False), ('cancellation_resolved', '=', True), ('state', '==', 'cancel')]}"
+                    string="Ignore backend's cancellation, Keep Open"
+                    groups="base.group_sale_manager" />
```
Comment (sebalix) [inline connector_ecommerce/views/sale_view.xml]:
```suggestion
                    groups="sales_team.group_sale_manager" />
```

File: connector_ecommerce/tests/test_picking_event.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        partner = partner_model.create({'name': 'Benjy'})
+        self.product_6 = self.env.ref('product.product_product_6')
+        self.product_7 = self.env.ref('product.product_product_7')
+        self.sale = self.sale_model.create({'partner_id': partner.id})
+        self.sale_line_model.create({
+            'order_id': self.sale.id,
+            'product_id': self.product_6.id,
+            'name': "iPad Mini",
+            'product_uom_qty': 42,
+            'product_uom': self.env.ref('product.product_uom_unit').id,
```
Comment (ivantodorovich) [inline connector_ecommerce/tests/test_picking_event.py]:
```suggestion
            'product_uom': self.env.ref('uom.product_uom_unit').id,
```

File: connector_ecommerce/tests/test_picking_event.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            'price_unit': 65,
+        })
+        self.sale_line_model.create({
+            'order_id': self.sale.id,
+            'product_id': self.product_7.id,
+            'name': "Apple In-Ear Headphones",
+            'product_uom_qty': 2,
+            'product_uom': self.env.ref('product.product_uom_unit').id,
```
Comment (ivantodorovich) [inline connector_ecommerce/tests/test_picking_event.py]:
```suggestion
            'product_uom': self.env.ref('uom.product_uom_unit').id,
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Comment by mlaitinen** [comment]
Update to the description: https://github.com/OCA/sale-workflow/pull/812 was closed because another PR containing the migrated `sale_exception` was merged at https://github.com/OCA/sale-workflow/pull/831. So that has been taken care of.

**Comment by mlaitinen** [comment]
@acm1pt-colorado Both of the PR's mentioned in the description are now merged.

**Comment by ivantodorovich** [comment]
@acm1pt-colorado can you please force travis/runbots to rerun?
both prs are merged now

**Review by sebalix — APPROVED** [review]

**Comment by ivantodorovich** [comment]
runbot is giving warnings:
```
2019-10-09 16:32:54,648 172 WARNING openerp_test odoo.models: exception.rule.create() with unknown fields: rule_group
2019-10-09 16:32:54,653 172 WARNING openerp_test odoo.models: exception.rule.create() with unknown fields: rule_group
2019-10-09 16:32:54,657 172 WARNING openerp_test odoo.models: exception.rule.create() with unknown fields: rule_group
```

`rule_group` was removed here: https://github.com/OCA/server-tools/commit/254391987cea07e93c668399ee1c6d51d8491fb2#diff-c7fd995df65e10a635951020066e82a6

**Comment by ivantodorovich** [comment]
@acm1pt-colorado looks like that last commit is authored by someone that hasn't signed the CLA
can you please sign it, or amend that last commit?

Besides that, everything looks good to me! :+1:  !

**Review by ivantodorovich — APPROVED** [review]
:+1: 

besides CLA, everything else LGTM

**Comment by ljimenezsidoo** [comment]
Thanks @ivantodorovich 

Im tried to see the error from runbot but im not able to saw the logs.

**Comment by ivantodorovich** [comment]
> Thanks @ivantodorovich
> 
> Im tried to see the error from runbot but im not able to saw the logs.

Hmmm. You fixed it here: https://github.com/OCA/connector-ecommerce/pull/49/commits/d515cdf3ffed143fb025fea46a747cc4800125fd

There's no more error :+1:

**Comment by ivantodorovich** [comment]
@acm1pt-colorado the only thing missing is this:

> Hey @acm1pt-colorado, thank you for your Pull Request.
> 
> It looks like some users haven't signed our **C**ontributor **L**icense **A**greement, yet.
> You can read and sign our full Contributor License Agreement here: http://odoo-community.org/page/cla
> Here is a list of the users:
> 
> * LuisAdan [luis.jimenez@pesol.es](mailto:luis.jimenez@pesol.es) (no github login found)
> 
> Appreciation of efforts,
> OCA CLAbot

**Comment by guewen** [comment]
@acm1pt-colorado thanks for your work, can you confirm that you signed the CLA?

**Comment by ljimenezsidoo** [comment]
Hi @guewen @ivantodorovich 

I did with the user acm1pt-colorado but i dont know why is asking me again for the user LuisAdan this is my gitlab user.

image

**Comment by guewen** [comment]
@acm1pt-colorado thanks, I checked in the odoo-community DB and I found your account with the ICLA tag.

/ocabot merge

**Comment by ivantodorovich** [comment]
Thanks for the hard work @acm1pt-colorado !!! :)

**Pull request merged**